Agents/EnvironmentExposureCalculationAgent/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Install the fenland-trajectory-agent agent in Docker container
+#==================================================================================================
+FROM ghcr.io/cambridge-cares/stack-client:1.40.1 as stackclients
+#------------------------------------------------------
+# Base image to be reused
+#------------------------------------------------------
+FROM python:3.9.14-slim-buster as base
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED=1
+
+# Install Java
+RUN apt update && apt install -y openjdk-11-jdk-headless git
+
+# Copy StackClients resource required to update twa later
+RUN mkdir -p ./tmp_stack
+COPY --from=stackclients /app ./tmp_stack
+
+# Set the default working directory & copy files to install Python
+WORKDIR /app
+COPY ./setup.py .
+# COPY ./resources/ontop.obda ./resources/ontop.obda
+ADD https://jdbc.postgresql.org/download/postgresql-42.5.0.jar ./resources/
+
+# Install Python package and the required libraries
+RUN python -m pip install --upgrade pip
+RUN pip install -e .
+# Installation of agentlogging currently suspended due to performance issues
+# Install StackClients twa resource
+RUN stack_clients_jar=$(find ../tmp_stack/stack-clients*.jar) && stack_clients_jar=${stack_clients_jar##*/} && \
+    jpsrm install StackClients ../tmp_stack --jar $stack_clients_jar
```

Comment:
I found that it is necessary to run `jpsrm uninstall StackClients` when the a new twa package is downloaded. Not critical but probably better to reduce the number of instances where this line is copied over, though redundant once a StackClients package is released properly